snap/snapcraft.yaml

```diff
@@ -1,7 +1,7 @@
 # yamllint disable rule:line-length
 name: charmed-postgresql # you probably want to 'snapcraft register <name>'
 base: core22 # the base snap is the execution environment for this snap
-version: '14.10' # just for humans, typically '1.2+git' or '1.3.2'
+version: '14.11' # just for humans, typically '1.2+git' or '1.3.2'
```

Comment:
Consider to delete those unrelated comments (here and above) the next ticket update.